include/triton/Dialect/TritonGPU/IR/Dialect.h

```diff
@@ -33,6 +33,13 @@ SmallVector<unsigned> getSizePerThread(Attribute layout);
 
 SmallVector<unsigned> getContigPerThread(Attribute layout);
```

Comment:
I think we've reached the point where we need comments describing each function with a small example. Things are no longer as self-explanatory as they were back when we only had a single `ThreadsPerWarp` for distributed layouts

Reply:
Yea I agree, I'll add comments